### PR TITLE
Update for xcode 10 changes

### PR DESCRIPTION
**Pull request by dostrander**
@dnkoutso @segiddins @amorde updates for xcode 10 default project settings

### CODE REVIEW

**Comment by dnkoutso** [comment]
@amorde didnt we do this?

**Comment by amorde** [comment]
Ahh, I updated Xcodeproj but never did it on the CocoaPods side

File: install_static_swift_modules/after/Pods/Headers/Public/MixedPod/MixedPod-library-macOS.modulemap [hunk 1]
```diff
@@ -1 +1 @@
-/Users/segiddins/Development/Rainforest/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/MixedPod-library-macOS/MixedPod-library-macOS.modulemap
\ No newline at end of file
+/Users/derko/Development/cocoapods/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/MixedPod-library-macOS/MixedPod-library-macOS.modulemap
```
Comment (dnkoutso) [inline install_static_swift_modules/after/Pods/Headers/Public/MixedPod/MixedPod-library-macOS.modulemap]: uh why do we have the absolute path here?

File: install_static_swift_modules/after/Pods/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap [hunk 2]
```diff
@@ -1 +1 @@
-/Users/segiddins/Development/Rainforest/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.modulemap
\ No newline at end of file
+/Users/derko/Development/cocoapods/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.modulemap
```
Comment (amorde) [inline install_static_swift_modules/after/Pods/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap]: could you undo these changes? its on our list of TODOs to make these use relative / mocked paths but for now would rather leave it at the current value

File: install_static_swift_modules/after/Pods/Headers/Public/MixedPod/MixedPod-library-macOS.modulemap (hunk 1 above)
Comment (amorde) [reply install_static_swift_modules/after/Pods/Headers/Public/MixedPod/MixedPod-library-macOS.modulemap]: https://cocoapods.slack.com/archives/C029NJ045/p1533581255000320
